File: Tests/CK.StObj.TypeScript.Tests/CrisLike/Model/ICommandAuthUnsafe.cs
#pragma warning disable CS1574 // XML comment has cref attribute that could not be resolved

namespace CK.CrisLike
{
    /// <summary>
    /// Defines the <see cref="ActorId"/> field.
    /// This is the most basic command part: the only guaranty is that the actor identifier is the
    /// current <see cref="IAuthenticationInfo.UnsafeUser"/>.
    /// </summary>
    public interface ICommandAuthUnsafe : ICommandPart
    {
        /// <summary>
        /// Gets or sets the actor identifier.
        /// The default <see cref="CrisAuthenticationService"/> validates this field against the
        /// current <see cref="IAuthenticationInfo.UnsafeUser"/>.
        /// </summary>
        [EndpointValue]
        int ActorId { get; set; }
    }

}

File: Tests/CK.TypeScript.Tests/TSInlineTests/CrisLike_commands_and_results/ck-gen/CK/CrisLike/ICommandAuthUnsafe.ts
import { ICommandPart } from './ICommandPart';

/**
 * Defines the ICommandAuthUnsafe.actorId field.
 * This is the most basic command part: the only guaranty is that the actor identifier is the
 * current ~~!:IAuthenticationInfo.UnsafeUser~~.
 **/
export interface ICommandAuthUnsafe extends ICommandPart {
/**
 * Gets or sets the actor identifier.
 * The default ~~!:CrisAuthenticationService~~ validates this field against the
 * current ~~!:IAuthenticationInfo.UnsafeUser~~.
 **/
actorId: number;
readonly _brand: ICommandPart["_brand"] & {"30":any};
}
